array_api_compat/dask/array/linalg.py

```diff
@@ -44,7 +52,7 @@
 def svd(x: Array, full_matrices: bool = True, **kwargs) -> SVDResult:
     if full_matrices:
         raise ValueError("full_matrics=True is not supported by dask.")
-    return da.linalg.svd(x, **kwargs)
+    return da.linalg.svd(x, coerce_signs=False, **kwargs)
```

Comment:
Needed sign dask will do a ``svd_flip`` on the data otherwise, which will make signs differ from numpy/scipy.